support-frontend/test/selenium/subscriptions/pages/CheckoutPage.scala

```diff
@@ -55,6 +56,12 @@ trait CheckoutPage extends Page with Browser {
     switchToFrame(2)
     setValue(cvc, "123")
     switchToParentFrame
+    clickOn(stripeRecaptchaButton)
+    waitForTestRecaptchaToComplete
+  }
+
+  def waitForTestRecaptchaToComplete: Unit = {
+    Thread.sleep(1000)
```

Comment:
Is there any way we could use some in page state here to determine whether the recaptcha is complete or not rather than hard coding 1 second?

There are some examples of things that might work here:
https://github.com/guardian/support-frontend/blob/842a0042ab5ed9d7994895db94b87604dafff5b6/support-frontend/test/selenium/util/Browser.scala#L36

Reply:
That seems sensible - will have a look into how this works now

Reply:
Tried doing this with `waitUntil(ExpectedConditions.not(ExpectedConditions.elementToBeClickable(q.by)))`, but did not work. Might leave it as a refactoring task for later on.